Backend/src/main/java/com/project/omega/controller/OrderController.java

```diff
@@ -101,6 +101,12 @@ public ResponseEntity fetchAllOrders() {
         return new ResponseEntity(order, HttpStatus.OK);
     }
 
+    @PutMapping(value = "/{id}")

```

Comment:
Change to "/update/{id}" pls. Otherwise all good.

Reply:
Updated now to conform